doc/user-guides/auth/auth-for-app-devs-and-platform-engineers.md

```diff
@@ -35,39 +35,12 @@ Topology:
             └─────────────────┘
 ```
 
-## Requisites
+### Setup the environment
 
-- [Docker](https://docker.io)
+Follow this [setup doc](https://github.com/Kuadrant/kuadrant-operator/blob/main/doc/install/install-make-target.md) to set up your environment before continuing with this doc.
 
-## Run the guide ① → ④
 
-### ①  Setup (Persona: _Cluster admin_)
-
-Clone the repo:
-
-```sh
-git clone git@github.com:Kuadrant/kuadrant-operator.git && cd kuadrant-operator
-```
-
-Run the following command to create a local Kubernetes cluster with [Kind](https://kind.sigs.k8s.io/), install & deploy Kuadrant:
-
-```sh
-make local-setup
-```
-
-Request an instance of Kuadrant in the `kuadrant-system` namespace:
-
-```sh
-kubectl -n kuadrant-system apply -f - <<EOF
-apiVersion: kuadrant.io/v1beta1
-kind: Kuadrant
-metadata:
-  name: kuadrant
-spec: {}
-EOF
-```
-
-### ② Deploy the Toy Store sample application (Persona: _App developer_)
+### Deploy the Toy Store sample application (Persona: _App developer_)
```

Comment:
Removing the unicode numbering to match the rest of the docs and to make writing docs easier